utils/zerovec/src/map/vecs.rs

```diff
@@ -121,9 +107,9 @@ pub trait ZeroVecLike<'a, T: ?Sized> {
 /// longer references to the underlying buffer, for borrowed-only vector types.
 ///
 /// Methods are prefixed with `zvl_*` to avoid clashes with methods on the types themselves
-pub trait BorrowedZeroVecLike<'a, T: ?Sized>: ZeroVecLike<'a, T> {
-    /// Get element at `index`, with a longer lifetime
-    fn zvl_get_borrowed(&self, index: usize) -> Option<&'a Self::GetType>;
+pub trait BorrowedZeroVecLike<T: ?Sized>: ZeroVecLike<T> {
+    /// Return a new, empty value with the static lifetime
+    fn zvl_new_borrowed() -> &'static Self;
```

Comment:
I could put this directly on `ZeroVecLike` and remove `BorrowedZeroVecLike` entirely. Opinions?